scribble/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,39 @@
+class CommentsController < ApplicationController
+  def index
+  end
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @post.comments.create!(comment_params)
+    redirect_to post_path(@post)
```

Comment:
Just a reminder that this can be shortened to `redirect_to @post`, not necessary, some prefer it, some don't!